jobs/base_model_validator.py

```diff
@@ -147,11 +169,16 @@ def expand(self, model_pipe):
                 ValidateModelIdWithRegex(), self._get_model_id_regex())
         )
 
+        commit_type_validation_errors = (
+            not_deleted | beam.ParDo(ValidateCommitType())
+        )
```

Comment:
Why is it added here, this is a generic base class for all models and all models do not have the commit type.

Reply:
I wrote it because of [this](https://github.com/oppia/oppia/pull/12240#discussion_r598202357) comment of @brianrodri . I think I misunderstood it. What does this comment actually mean. Thanks!

Reply:
Ah, I meant most-common class. So if BaseSnapshotMetadataModelValidator is the most common class with this property, we should write a validator that's specific to it.

Reply:
Done